### ISSUE TITLE
overwrite message while installing OSX Mountain Lion

### ISSUE THREAD

**Issue by rdymac** [maran/mastercoin-wallet#5]
```
.
.
mastercoin-wallet's executable "console" conflicts with mastercoin-ruby
Overwrite the executable? [yN]  
.
.
```

Correct answer: yes

**Comment by bitcoiners** [comment]
Same issue installing on Ubuntu x64 13.10

What is correct answer yes or no?

**Comment by rdymac** [comment]
@slavix the correct answer is Yes. It will be fixed in the future.

**Comment by maran** [comment]
This should be fixed in 0.0.13 :)
